src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -94,18 +98,24 @@ public OrganizationUsersController(
         _twoFactorIsEnabledQuery = twoFactorIsEnabledQuery;
         _removeOrganizationUserCommand = removeOrganizationUserCommand;
         _deleteManagedOrganizationUserAccountCommand = deleteManagedOrganizationUserAccountCommand;
+        _getOrganizationUsersManagementStatusQuery = getOrganizationUsersManagementStatusQuery;
+        _featureService = featureService;
     }
 
     [HttpGet("{id}")]
-    public async Task<OrganizationUserDetailsResponseModel> Get(string id, bool includeGroups = false)
+    public async Task<OrganizationUserDetailsResponseModel> Get(Guid id, bool includeGroups = false)
```

Comment:
Can you point me to the UI change that requires this model to be updated? As far as I can see, it's used in the member dialog and to confirm a user, but I couldn't see any Figma designs updating those locations.